Car.h

```diff
@@ -1,9 +1,13 @@
 #pragma once
 
+#include "Vehicle.h"
+
 struct Car : public Vehicle
 {
     Car(const std::string& s);
-    void closeWindows();
 
+    void closeWindows();
+    
     void tryToEvade();
-};
\ No newline at end of file
+    void setSpeed(int);
```

Comment:
- [ ] remove this declaration.

Reply:
Bit confused about why I can remove this declaration of what is a virtual function in the Base class from Car.h , but I can't do it from `SemiTruck.h` without many `-weakvtable` warnings?  I needed to implement overidden declarations with empty out-of-line definitions in `SemiTruck.h` and `SemiTruck.cpp` to clear the warnings.... but not so in `Car.h` ? Why?